mathics/builtin/strings.py

```diff
@@ -2488,8 +2489,8 @@ class StringInsert(Builtin):
      = abcdefghijklm
 
     #> StringInsert["", "", 1]
-     =
-
+     = 
+    (watch the empty line).
```

Comment:
It is inteded to be a reminder that there needs to be an empty space the end of 2492. My editor by default removes that. 

I got this wording from some other place in the code. 

Down the line I will fix testing so that we can wrote something like.

```
= (* no output *)
```

In fact that may work right now. If it works, I'd like to start adding those so people (like me or people who use Emacs) don't trip over this.